package.json

```diff
@@ -363,6 +363,11 @@
           "default": null,
           "description": "Specifies the full path to the OmniSharp server."
         },
+        "omnisharp.experimentalOmnisharp": {
+          "type": "string",
+          "default": null,
+          "description": "Specifies if the user wants to use the experiment build. Possible values - latest or some version number"
+        },
```

Comment:
Please provide "enum" values to help with JSON completion and schema validation. See "omnisharp.loggingLevel" below for an example.

Reply:
Also, please clean up the grammar in the description. This is a user-facing string.

Reply:
Also, **omnisharp**.experimental**OmniSharp** is a bit redundant. It's also not clear to me why this is "experimental". Maybe `omnisharp.alternateVersion` or something like that? @rchande?